packages/gitbook/src/components/Header/HeaderLink.tsx

```diff
@@ -52,20 +65,20 @@ export async function HeaderLink(props: {
                 })();
                 return (
                     <Button
-                        href={target.href}
+                        href={target?.href}
                         variant={variant}
                         size="medium"
                         className={tcls(
                             {
-                                'button-primary':
+                                ['button-primary']:
```

Comment:
just curious, why the change with the brackets?

Reply:
It was just a change and another change.